dubbo-config/dubbo-config-api/src/main/java/org/apache/dubbo/config/ServiceConfig.java

```diff
@@ -145,6 +145,11 @@
      */
     private ProviderConfig provider;
 
+    /**
+     * The metrics configuration
+     */
+    private MetricsConfig metrics;

```

Comment:
Both ServiceConfig and ReferenceConfig have this property, how about moving it to one shared superclass.